source/codegen/templates/service.cpp.mako

```diff
@@ -100,6 +111,20 @@ namespace ${config["namespace_component"]}_grpc {
   }
 
 % endif
+% if streaming_functions_to_generate:
+void RegisterMonikerEndpoints()
```

Comment:
This needs to be indented one more level (same as `CopyBytesToEnums` function above). And the method body will then indent one more level.

Reply:
Same for `define_moniker_function_body` as well.